src/main/java/organice/logic/commands/DoneCommand.java

```diff
@@ -35,25 +37,33 @@
     private Nric donorNric;
 
     private String statusDone = "done";
+    private String statusNotProcessing = "not processing";
+    private String result;
 
     /**
-     * creates a DoneCommand to remove the given patient and donor pair
+     * Creates a DoneCommand to determine if the given patient and donor can be deleted from the system.
+     * If the cross-matching fails, the patient and donor will need to be in the system for further matching.
+     * If the cross-matching pass, the patient and donor can be deleted from ORGANice.
      * @param firstNricString
      * @param secondNricString
      */
-    public DoneCommand(String firstNricString, String secondNricString) {
+    public DoneCommand(String firstNricString, String secondNricString, String result) {
         requireNonNull(firstNricString, secondNricString);
         firstNric = new Nric(firstNricString);
         secondNric = new Nric(secondNricString);
+        this.result = result;
     }
 
     /**
-     * check if the two Nric given are valid.
-     * It needs to contain one patient and one donor and both of them must be matched in order to be valid
-     * @param firstNric the first Nric given
-     * @param secondNric the second Nric given
+     * Method to check if the two Nrics given are valid.
```

Comment:
Suggestion to change the comment to this:
This method checks if the two Nrics given are of a valid patient-donor pair. 
The pair of Nrics is valid if one of the Nrics belongs to a patient in ORGANice and the other Nric belongs to a donor in ORGANice. 

(Btw, I see that your Nric inputs are no longer String objects?)

Reply:
amended